### PR TITLE
Add MVCCGet benchmark.

### PR DESCRIPTION
**Pull request by petermattis**
BenchmarkMVCCGet1Version     3000000  3956 ns/op  258.85 MB/s
BenchmarkMVCCGet10Versions   3000000  4392 ns/op  233.11 MB/s
BenchmarkMVCCGet100Versions  3000000  5182 ns/op  197.58 MB/s

### CODE REVIEW

File: storage/engine/rocksdb_test.go [hunk 1]
```diff
@@ -297,6 +296,52 @@ func BenchmarkMVCCScan100Versions1000Rows(b *testing.B) {
...
+
+	b.RunParallel(func(pb *testing.PB) {
+		for pb.Next() {
+			// Choose a random key to retrieve.
+			keyIdx := rand.Int31n(int32(numKeys))
+			key := proto.Key(encoding.EncodeVarUint32([]byte("key-"), uint32(keyIdx)))
+			walltime := int64(5 * (rand.Int31n(int32(numVersions)) + 1))
+			ts := makeTS(walltime, 0)
+			_, err := MVCCGet(rocksdb, key, ts, nil)
+			if err != nil {
```
Comment (spencerkimball) [inline storage/engine/rocksdb_test.go]: Not like it matters but unless you need the results, better to combine these two statements into a single if.

File: storage/engine/rocksdb_test.go [hunk 2]
```diff
@@ -297,6 +296,52 @@ func BenchmarkMVCCScan100Versions1000Rows(b *testing.B) {
...
+	// preferential treatment to tests with fewer keys. Note that the
+	// datasets all fit in cache and the cache is pre-warmed.
+	const numKeys = 100000
+
+	rocksdb := setupMVCCScanData(numVersions, numKeys, b)
+	defer rocksdb.Stop()
+
+	prewarmCache(rocksdb)
+
+	b.SetBytes(1024)
```
Comment (spencerkimball) [inline storage/engine/rocksdb_test.go]: Is 1024 correct here? Is this the number of nominal bytes read per get?

**Comment by spencerkimball** [comment]
LGTM

File: storage/engine/rocksdb_test.go (hunk 1 above)
Comment (petermattis) [inline storage/engine/rocksdb_test.go]: Done.

File: storage/engine/rocksdb_test.go (hunk 2 above)
Comment (petermattis) [inline storage/engine/rocksdb_test.go]: This is the size of values created in `setupMVCCScanData`. It doesn't account for the key sizes, but I think that is fine.

**Pull request merged**